### DISCUSSION TITLE
Serenity Core TabExtention TSX

### DISCUSSION THREAD

**Discussion by yayaya23** [serenity-is/Serenity#7196] [Help]
Hi, i'm starting a project with serenity core using serenity corelib 8.5.6

Previously i was using Serenity framework, i built the dialog with entity grid inside. it was using common html template with some default prefix and its perfectly  running. And now I want to consume the same thing with serenity core. I follow the code like example https://github.com/serenity-is/common-features/blob/master/src/Serenity.Demo.Northwind/Modules/Customer/CustomerDialog.tsx.

Unfortunately i isn't running in my code. the problem is the object "this.ordersGrid" is always getting undefined. is any one have experience regarding the issue ? here is my code
![DialogWorking](https://github.com/serenity-is/Serenity/assets/62537274/db259683-ca79-4576-94b3-f8fc32fa7855)

and here is the grid
![Grid need to put in dialog](https://github.com/serenity-is/Serenity/assets/62537274/f297cde6-9e3d-44ee-bcba-418d769438b9)

Very appreciate your help !!!!!!

**Comment by volkanceylan** [comment]
You should have "useDefineForClassFields": false in tsconfig.json under compilerOPptions
